lib/hashes.md

```diff
@@ -5,40 +5,53 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A dictionary like collection of data that pairs unique keys to their corresponding vaules.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+Keys can be any data type.
+Strings and numbers would be the two most common data types. 
+
 
 # How are Hashes and Arrays similar? How are they different?
 
-Replace me with your response...
+Hashes are special arrays.  
+They can be used interchangably, but hashes are easier to maintain and manipulate.
```

Comment:
You said:

> They can be used interchangeably 

I think adding to your answer to the question above will help clarify instances when Hashes might be more appropriate to use than an Array. 

Let's say I want to collect a list of students. Let's also say, for each student, I want to collect information like their name, email, and age. Which collection would be most-appropriately stored as a Hash and which as an Array? Why?